docs/kandel/how-does-kandel-work/parameters.md

```diff
@@ -20,6 +20,6 @@ Initial inventory | The initial inventory is the amount of base tokens and quote
 Current price | The current price of the base token that is used for constructing the price distribution.<br /><br />*Example: the price of ETH is used for the ETH/USDC pair*
 Price distribution | It is a distribution of price levels/grid points within a selected price range. Kandel starts from the *Min price*, all the way up to the *Max price*. It is calculated using geometric progression and a key metric called "ratio": <br /><br />*Price at index i = Min_price x Ratio ^ i*
 Volume distribution | It defines how liquidity is spread throughout the price grid (previously defined as the price distribution in the [visual step-by-step](./step-by-step-visual-explanation))
-Step size | It is the distance between an executed bid/ask and its dual offer. In the case of an offer is taken at price p :<br />• Lowest price = 1000 USDC per ETH<br />• Highest price = 1500 USDC per ETH<br /><br />*Example: if step size=1, it means that if a bid is taken at an index i, the ask ([dual offer](../../terms/dual-offer.md)) is posted at an index i+1*
+Step size |  It is the distance between an executed bid/ask and its [dual offer](../../terms/dual-offer.md).<br /><br />Whenever a Kandle ask is taken at a given price point, Kandle uses the amount of quote just received to place a bid at a lower price point. With a step size of 1, it will place the bid at the price point immediately below. With a step size of 2, Kandel will repost two price points below, etc. The same applies symmetrically for bids.

```

Comment:
```suggestion
Step size |  It is the distance between an executed bid/ask and its [dual offer](../../terms/dual-offer.md).<br /><br />Whenever a Kandel ask is taken at a given price point, Kandel uses the amount of quote just received to place a bid at a lower price point. With a step size of 1, it will place the bid at the price point immediately below. With a step size of 2, Kandel will repost two price points below, etc. (Technical aside: if in attempting to repost say 3 steps below Kandel hits the boundaries of its range it will transport as far below as possible). The same applies symmetrically for bids.
Using a step size ≥ 2 allows one to publish a more continuous liquidity on the books,
regularising the strat’s PnL, while at the same time keeping a reasonable spread between price points. Indeed, what matters to PnL is not the distance between price points, but how far money moves along the price grid each time an offer is taken. 


```